opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/instrumentor.py

```diff
@@ -52,7 +52,9 @@ def instrument(self, **kwargs):
             self._is_instrumented = True
             return result
 
-        _LOG.warning("Attempting to instrument while already instrumented")
+        _LOG.warning(
```

Comment:
Is this change intentional?

Reply:
Must have slipped from other PR, reverting.